### DISCUSSION TITLE
Compass heading of crosshair in oculars plug-in

### DISCUSSION THREAD

**Discussion by HenryWede** [Stellarium/stellarium#2670] [Ideas]
I am using Stellarium as a night photographer and not an astronomer, so I hope this doesn't sound too silly.
It would be helpful for me to show the position of the center cross in terms of a compass heading instead of the RA/Dec that is currently shown.  For example, 220 degrees would be a lot more helpful for me.  I can match it up to my GPS, measure it in Google Earth, and so on.

Are the plug-ins written in such a way that I could edit a couple lines (with somebody's help) to add this?  Or would the authors be willing to do this?  

The attached image shows how the compass heading can be hard to capture.  The rectangle is showing the FOV for a 16mm lens.

Thank you for your help.
![image](https://user-images.githubusercontent.com/22160924/191132734-c6c74327-b269-407d-aa92-c62200ab552d.png)

**Comment by alex-w** [comment]
We may add optional support horizontal coordinates for the CCD sensor

**Comment by alex-w** [comment]
Hello @HenryWede!

The enhancement has been implemented! You may test it via building Stellarium from source code or wait the weekly development snapshot...

**Reply by HenryWede** [reply]
Wow!  Thank you so much Alex.  I'll wait for a week or two and then grab the latest release.
I should always expect this kind of prompt positive response to all my suggestions, right?  :)

**Reply by gzotti** [reply]
Just the easy ones...

**Comment by alex-w** [comment]
Hello @HenryWede!

Please check the fresh version (development snapshot) of Stellarium:
https://github.com/Stellarium/stellarium-data/releases/tag/weekly-snapshot
